visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.scss

```diff
@@ -30,26 +31,77 @@
 	}
 }
 
+.ai-suspicious-metrics-list {
+	display: flex;
+
+	.metric-button {
+		min-height: 36px;
+		line-height: 23px;
+		min-width: 45px;
+		padding: 0 12px 0 12px;
+		text-transform: none;
+		margin-right: 0;
+		overflow: visible;
+	}
+
+	.risk-button {
+		margin-left: 0;
+		min-width: 36px;
+		width: 36px;
+		height: 36px;
+		color: #9c0e6a;
+	}
+}
+
+.ai-drop-down {
+	padding: 0;
+}
+
+md-button.title {
+	color: rgba(0, 0, 0, 0.38);
+	flex-direction: column;
+}
+
+md-menu-item.title-item {
+	height: 100%;
+
+	p.title {
+		margin: 12px 0px 12px;
+		line-height: 24px;
+	}
+}
+
 md-menu-item.ai-list-item {
 	padding: 0 16px;
+	margin-bottom: 10px;
+	height: auto;
+	min-height: unset;
 }
 
 md-menu-item.risk-profile {
+	margin: 0 12px 12px 12px;
+
 	.md-button {
+		text-align: center;
+		color: black !important;
```

Comment:
!important is evil. Never ever use it, just try to make a more specific class or selector.

Reply:
Yes, but in that place it seems necessary. Because we need to override md properties. If you have another way, let us know

Reply:
```suggestion
		color: black;
		padding: 0;
		margin: auto 0;

    &.low-risk,
    &.moderate-risk,
    &.high-risk {
      color: black;
    }
```